calculator-server/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+const router = require('../routes/calculatorWebPage')
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const port = 3000
+
+app.use(cors())
+app.use(bodyParser.json())
+app.use('/api/v1/calculator', router)
+app.listen(port, console.log('Server is up'))
```

Comment:
You're technically calling `Server is up` before calling `app.listen`, let alone before `app.listen` actually finishes.

Also what if there's an error listening? You're still logging "server is up".